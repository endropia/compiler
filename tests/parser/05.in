var
a, b : Integer = 10;
begin
end.