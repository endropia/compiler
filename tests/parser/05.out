(2, 16) Only ine variable can be initialized